test/runner/reporter.js

```diff
@@ -3,6 +3,15 @@
 import { prettyMs } from "./lib/prettyMs.js";
 import * as Diff from "diff";
 
+const rdquo = /"/g;
+
+function quoteWrap( value ) {
+	if ( typeof value === "string" ) {
+		return `"${ value.replace( rdquo, "\\\"" ) }"`;
```

Comment:
@timmywil I think this alert was applicable and we need to properly escape at least some backlash characters as well here...

Reply:
Do you have a case in mind? I can't think of any.

Reply:
`quoteWrap('\\"')` would return `'"\\\\""'`. If you try to evaluate the value, i.e. `"\\\\""`, you will get a syntax error since `\\\\` is actually two backslashes and the last but one `"` is effectively unescaped.

Basically, if we escape a quote by prepending it with a backslash, an existing backslash in the input string will lead to nonsensical output.

Reply:
My goal is to show that one value is a string while the other is not, (i.e. to differentiate between `"undefined"` and `undefined`. But, I wasn't concerned about syntax errors from copying/pasting console output when including the extra quotes, as long as there was no syntax error when printing it, which is not the issue here (and therefore is not a security issue).

I interpret the outer quotes as discrete characters and never cared if there were other double quotes in the string. In other words, I would never include the other quotes in my copying. It's similar to printing strings as a different color, but you can't see that in github actions.

That said, I don't want to keep going around in circles, so I'd just as soon give up my quest to differentiate `"undefined"` and `undefined`.